File: src/main/java/infinitespire/avhari/Avhari.java
package infinitespire.avhari;

import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import com.megacrit.cardcrawl.characters.AnimatedNpc;
import com.megacrit.cardcrawl.core.Settings;
import infinitespire.InfiniteSpire;

public class Avhari {
	AnimatedNpc anim;

	public Avhari(float x, float y) {
		anim = new AnimatedNpc(
			(Settings.WIDTH / 2f) - (150f * Settings.scale),
			450f * Settings.scale,
			InfiniteSpire.createPath("ui/avhari/character/Avhari.atlas"),
			InfiniteSpire.createPath("ui/avhari/character/Avhari.json"),
			"idle");
		anim.skeleton.getRootBone().setScale(0.5f);
	}

	public void render(SpriteBatch sb) {
		anim.render(sb);
	}
}

File: src/main/kotlin/com/blanktheevil/infinitespire/rooms/avhari/Avhari.kt
package com.blanktheevil.infinitespire.rooms.avhari

import com.badlogic.gdx.graphics.g2d.SpriteBatch
import com.blanktheevil.infinitespire.interfaces.SpireClickable
import com.blanktheevil.infinitespire.interfaces.SpireElement
import com.megacrit.cardcrawl.helpers.Hitbox

class Avhari : SpireElement, SpireClickable {
  private val hb = Hitbox(0f,0f)

  override fun getHitbox(): Hitbox = hb

  override fun update() {
  }

  override fun render(sb: SpriteBatch) {
  }
}